finanse/cashflow_plot.html

```diff
@@ -0,0 +1,2 @@
+<div>                        <script type="text/javascript">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>

```

Comment:
Możesz to wrzucić bezpośrednio do `finanse/index.html` zaraz pod `<h2>Przepływ</h2>` zamiast tego co tam teraz siedzi?

Reply:
Poprawiłem. Przy odpaleniu skryptu z nowymi danymi będzie podmieniał fragment html w finanse/index.html 
Dodany jest też podmieniony już html z wykresem.